resources/js/app.js

```diff
@@ -1 +1 @@
-require('./bootstrap');
```

Comment:
@sjardim out of curiosity, why did you remove this line?

Reply:
For this PR in case, I should have not removed it, my bad! 
But I usually do it when it's not needed.